### PR TITLE
6956 - Adjusted the table's width to fill the container of the datagrid, ensuring proper alignment

### LINKED ISSUE #6956: Modal: Whitespace on the right side of a datagrid on Firefox
**Describe the bug**
In the Firefox browser, when a modal with a datagrid is opened there is some whitespace next to the last column in the datagrid.

**To Reproduce**
1. Go to http://localhost:4000/components/modal/test-modal-datagrid-title-overlap.html in Firefox
2. Click on Open Modal button
3. See that there is whitespace on the right side of the datagrid

**Expected behavior**
The columns should fill the entire datagrid.

**Version**
- ids-enterprise: v4.67.4

**Screenshots**
image

**Platform**
 - Infor Application/Team Name: lawson
 - OS Version: Windows 11
 - Browser Name: firefox
 - Browser Version: 107.0b9 (32-bit)

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR fixes the table's width to fill the container of the datagrid in firefox.

**Related github/jira issue (required)**:

Closes #6956

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Open the test page in `firefox` http://localhost:4000/components/modal/test-modal-datagrid-title-overlap.html
- Click on `Open Modal` button
- Table should filled the container of the datagrid (no whitespaces)

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
- [x] A note to the change log.

### CODE REVIEW

File: src/components/datagrid/_datagrid.scss [hunk 1]
```diff
@@ -4715,6 +4715,10 @@ td .btn-actions {
       }
     }
   }
+
+  .modal table {
+    min-width: -moz-available;
```
Comment (tmcconechy) [inline src/components/datagrid/_datagrid.scss:4720]: This work on chrome?

**Review by tmcconechy — DISMISSED** [review]

**Review by deep7102 — DISMISSED** [review]

**Review by janahintal — CHANGES_REQUESTED** [review]
Also experiencing the unnecessary scroll bar in Chrome and FF. 
image

**Comment by ericangeles** [comment]
I have tested it on both Chrome and Firefox in Windows, and I can confirm that both issues have been resolved. It is possible that you do not have the latest version of the code @janahintal.

**Review by jbrcna — COMMENTED** [review]
the table overflows in responsive view and landscape mobile
adjust the height to below 630 or switch to landscape mobile view
image

**Comment by s-werking** [comment]
I am also encountering the overflow issue that @jbrcna is seeing. Otherwise I no longer see the issue, thanks!

**Comment by tmcconechy** [comment]
@ericangeles can you check the last couple things. Would like to patch these

**Review by tmcconechy — APPROVED** [review]

**Comment by tmcconechy** [comment]
look good now @janahintal ?

**Comment by tmcconechy** [comment]
@s-werking this look ok enough for you folks?

**Comment by s-werking** [comment]
Looks good to me, thanks!

**Pull request merged**